G1ANT.Addon.GoogleDocs/G1ANT.Addon.GoogleDocs.Tests/Commands/googlesheet.getvalue/ShouldGetValue_WhenSelectingByRange.G1ANT

```diff
@@ -0,0 +1,13 @@
+﻿addon googledocs version 2.205.17304.1000
+addon core version 4.99.18340.1354
+addon language version 4.99.18340.1354
+
+♥FileID = ‴147EH2vEjGVtbzzkT6XaI0eNZlY5Ec91wlvxN3HC4GMc‴
+♥expectedValue = ‴Andrew‴
```

Comment:
If you expect some values in the test data then you should store it also offline or give some description of how the test data should look like (available offline) or create it during the test.

Reply:
google drive is not working offline

Reply:
Key word: "also" :P